mitodl/open-discussions
# Remove unpublished OCW courses from search index

#### Pre-Flight checklist



#### What are the relevant tickets?
Closes #3561 

#### What's this PR do?
Modifies the `WebhookOCWNextView` code to remove unpublished OCW-Next courses from the search index.

#### How should this be manually tested?
- Find the `run_id` an existing OCW-Next course run in your local instance, import one if necessary.
- Run `curl -X POST -d  '{"webhook_key": "<settings.OCW_NEXT_SEARCH_WEBHOOK_KEY>","unpublished": true,"site_uid": "<run_id>","version": "live"}'
- Verify that the run's `Course.published` value is `False` and the course  no longer shows up in search results.

#### Any background context you want to provide?
Related to https://github.com/mitodl/ocw-studio/issues/1164 and its linked RFC
